client/selectors/locks.ts

```diff
@@ -116,7 +116,7 @@ export const workqueueItems = createSelector<
                 ...newItems.planning,
             ],
             (i) => i.lock_time
-        )
+        ).filter((x) => x.lock_time != null)
```

Comment:
I'm still unsure about this fix. It seems that it fixes a symptom and not the cause of the bug. How/when does a lock not have a `lock_time` attribute.

Reply:
I'm not sure whether we're just getting the wrong data, here's the request that's fetching the locks:

<img width="1506" alt="image" src="https://github.com/user-attachments/assets/795f5765-1ead-4987-9674-b70345de7645">

```
curl 'https://bel-uat.test.superdesk.org/api/planning_locks' \
  -H 'authority: bel-uat.test.superdesk.org' \
  -H 'accept: */*' \
  -H 'accept-language: en-GB,en-US;q=0.9,en;q=0.8' \
  -H 'authorization: Basic MDBlNmQ2MGQtMGY3MS00YzQ4LWEyYTQtMTM2Zjg1MzRlYzM4Og==' \
  -H 'content-type: application/json' \
  -H 'origin: http://localhost:9000' \
  -H 'referer: http://localhost:9000/' \
  -H 'sec-ch-ua: "Not_A Brand";v="8", "Chromium";v="120"' \
  -H 'sec-ch-ua-mobile: ?0' \
  -H 'sec-ch-ua-platform: "macOS"' \
  -H 'sec-fetch-dest: empty' \
  -H 'sec-fetch-mode: cors' \
  -H 'sec-fetch-site: cross-site' \
  -H 'user-agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/120.0.0.0 Safari/537.36' \
  --compressed
```

I'm doing the filtering later down the line, because there's no actual lock on the item itself. Either data coming from locks request is incorrect or we're missing some afterwards